config/puma/production.rb

```diff
@@ -19,6 +19,17 @@
 #
 workers ENV.fetch('FOREMAN_PUMA_WORKERS', 2).to_i
 
+# Puma worker timeout
+#
+# Verifies that all workers have checked in to the master process within
+# the given timeout. If not the worker process will be restarted. This is
+# not a request timeout, it is to protect against a hung or dead process.
+# Setting this value will not protect against slow requests.
+#
+# The minimum value is 6 seconds, the default value is 60 seconds
+#
+worker_timeout ENV.fetch('FOREMAN_PUMA_WORKER_TIMEOUT', 60).to_i
```

Comment:
```suggestion
unless ENV.fetch('FOREMAN_PUMA_WORKER_TIMEOUT', nil).nil?
  worker_timeout ENV.fetch('FOREMAN_PUMA_WORKER_TIMEOUT').to_i
end
```

Is it better to do it this way to avoid the hard-coded default? @parthaa @ekohl